### PR TITLE
Add mx5160 keyboard

### PR DESCRIPTION
**Pull request by Maartenwut**
## Description

Adds the mx-5160 PCB to the QMK repository.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/maartenwut/mx5160/readme.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# mx5160
+
+![mx5160](https://i.imgur.com/XZ9xUeq.jpg)
+
+## Support
+Keyboard Maintainer: [Maarten Dekkers](https://github.com/maartenwut)  
+Hardware Supported: mx-5160 (Chicony 5160c case)
```
Comment (mechmerlin) [inline keyboards/maartenwut/mx5160/readme.md]:
```suggestion
Hardware Supported: mx-5160 (Chicony 5160c case)  
```

Need extra spaces for markdown to render properly

File: keyboards/maartenwut/mx5160/readme.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Hardware Availability: [Github](https://github.com/Maartenwut/mx-5160)  
+
+## Build
+Make example for this keyboard (after setting up your build environment):
+
+    make maartenwut/mx5160:default
+
+See [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) then the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information.
```
Comment (mechmerlin) [inline keyboards/maartenwut/mx5160/readme.md]:
```suggestion
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

We have an updated blurb

File: keyboards/maartenwut/mx5160/rules.mk [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+#     processor frequency in Hz. You can then use this symbol in your source code to
+#     calculate timings. Do NOT tack on a 'UL' at the end, this will be done
+#     automatically to create a 32-bit value in your source code.
+#
+#     This will be an integer division of F_USB below, as it is sourced by
+#     F_USB after it has run through any CPU prescalers. Note that this value
+#     does not *change* the processor frequency - it should merely be updated to
+#     reflect the processor speed set externally so that the code can use accurate
+#     software delays.
+F_CPU = 16000000
```
Comment (mechmerlin) [inline keyboards/maartenwut/mx5160/rules.mk]:
```suggestion

```

You can take out lines 4-15

File: keyboards/maartenwut/mx5160/rules.mk [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+#     clock generation for the USB and other AVR subsections. Do NOT tack on a 'UL'
+#     at the end, this will be done automatically to create a 32-bit value in your
+#     source code.
+#
+#     If no clock division is performed on the input clock inside the AVR (via the
+#     CPU clock adjust registers or the clock division fuses), this will be equal to F_CPU.
+F_USB = $(F_CPU)
+
+# Interrupt driven control endpoint task(+60)
+OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
```
Comment (mechmerlin) [inline keyboards/maartenwut/mx5160/rules.mk]:
```suggestion

```
You can take out lines 18 thru 38

File: keyboards/maartenwut/mx5160/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+#include QMK_KEYBOARD_H
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+
+#define _MA 0 //Main layer
+#define _FN 1 //Main layer
+#define _____ KC_NO
```
Comment (noroadsleft) [inline keyboards/maartenwut/mx5160/keymaps/default/keymap.c]: QMK defines an alias for this keycode automatically.

https://github.com/qmk/qmk_firmware/blob/de5cadd63669119a6ba83757780dc66bb8f2816c/quantum/quantum_keycodes.h#L29-L31

If you use the built-in one – `s/_____/XXXXXXX/g` – you don't need this definition.

File: keyboards/maartenwut/mx5160/rules.mk [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+# If you don't know the bootloader type, then you can specify the
+# Boot Section Size in *bytes* by uncommenting out the OPT_DEFS line
+#   Teensy halfKay      512
+#   Teensy++ halfKay    1024
+#   Atmel DFU loader    4096
+#   LUFA bootloader     4096
+#   USBaspLoader        2048
+# OPT_DEFS += -DBOOTLOADER_SIZE=4096
+
+
```
Comment (noroadsleft) [inline keyboards/maartenwut/mx5160/rules.mk]: Lines 50-60 should be deleted.

File: keyboards/maartenwut/mx5160/rules.mk [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support (+2400 to 4200, depending on config)
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs (+400)
```
Comment (noroadsleft) [inline keyboards/maartenwut/mx5160/rules.mk]: Lines 64-80 should be:

```Makefile
BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
MIDI_ENABLE = no            # MIDI support
UNICODE_ENABLE = no         # Unicode
BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
AUDIO_ENABLE = no           # Audio output on port C6
FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
```

File: keyboards/maartenwut/mx5160/info.json [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
..., {"x":10, "y":1}, {"x":11, "y":1}, {"x":12, "y":1}, {"x":13, "y":1}, {"x":14, "y":1}, {"x":15, "y":1}, {"x":16, "y":1, "w":1.5}, {"x":18, "y":1}, {"x":19, "y":1}, {"x":20, "y":1}, {"x":21, "y":1}, {"x":0, "y":2}, {"x":1, "y":2}, {"x":2.5, "y":2, "w":1.75}, {"x":4.25, "y":2}, {"x":5.25, "y":2}, {"x":6.25, "y":2}, {"x":7.25, "y":2}, {"x":8.25, "y":2}, {"x":9.25, "y":2}, {"x":10.25, "y":2}, {"x":11.25, "y":2}, {"x":12.25, "y":2}, {"x":13.25, "y":2}, {"x":14.25, "y":2}, {"x":15.25, "y":2}, {"x":16.25, "y":2, "w":1.25}, {"x":18, "y":2}, {"x":19, "y":2}, {"x":20, "y":2}, {"x":21, "y":2}, {"x":0, "y":3}, {"x":1, "y":3}, {"x":2.5, "y":3, "w":1.25}, {"x":3.75, "y":3}, {"x":4.75, "y":3}, {"x":5.75, "y":3}, {"x":6.75, "y":3}, {"x":7.75, "y":3}, {"x":8.75, "y":3}, {"x":9.75, "y":3}, {"x":10.75, "y":3}, {"x":11.75, "y":3}, {"x":12.75, "y":3}, {"x":13.75, "y":3}, {"x":14.75, "y":3, "w":1.75}, {"x":16.5, "y":3}, {"x":18, "y":3}, {"x":19, "y":3}, {"x":20, "y":3}, {"x":21, "y":3, "h":2}, {"x":0, "y":4}, {"x":1, "y":4}, {"x":2.5, "y":4, "w":1.5}, {"x":5, "y":4, "w":1.5}, {"x":6.5, "y":4, "w":7}, {"x":13.5, "y":4, "w":1.5}, {"x":16, "y":4, "w":1.5}, {"x":18, "y":4}, {"x":19, "y":4}, {"x":20, "y":4}]
```
Comment (noroadsleft) [inline keyboards/maartenwut/mx5160/info.json]: Numpad Enter is out of sequence here.

```suggestion
            "layout": [{"x":0, "y":0}, {"x":1, "y":0}, {"x":2.5, "y":0}, {"x":3.5, "y":0}, {"x":4.5, "y":0}, {"x":5.5, "y":0}, {"x":6.5, "y":0}, {"x":7.5, "y":0}, {"x":8.5, "y":0}, {"x":9.5, "y":0}, {"x":10.5, "y":0}, {"x":11.5, "y":0}, {"x":12.5, "y":0}, {"x":13.5, "y":0}, {"x":14.5, "y":0}, {"x":15.5, "y":0}, {"x":16.5, "y":0}, {"x":18, "y":0}, {"x":19, "y":0}, {"x":20, "y":0}, {"x":21, "y":0}, {"x":0, "y":1}, {"x":1, "y":1}, {"x":2.5, "y":1, "w":1.5}, {"x":4, "y":1}, {"x":5, "y":1}, {"x":6, "y":1}, {"x":7, "y":1}, {"x":8, "y":1}, {"x":9, "y":1}, {"x":10, "y":1}, {"x":11, "y":1}, {"x":12, "y":1}, {"x":13, "y":1}, {"x":14, "y":1}, {"x":15, "y":1}, {"x":16, "y":1, "w":1.5}, {"x":18, "y":1}, {"x":19, "y":1}, {"x":20, "y":1}, {"x":21, "y":1}, {"x":0, "y":2}, {"x":1, "y":2}, {"x":2.5, "y":2, "w":1.75}, {"x":4.25, "y":2}, {"x":5.25, "y":2}, {"x":6.25, "y":2}, {"x":7.25, "y":2}, {"x":8.25, "y":2}, {"x":9.25, "y":2}, {"x":10.25, "y":2}, {"x":11.25, "y":2}, {"x":12.25, "y":2}, {"x":13.25, "y":2}, {"x":14.25, "y":2}, {"x":15.25, "y":2}, {"x":16.25, "y":2, "w":1.25}, {"x":18, "y":2}, {"x":19, "y":2}, {"x":20, "y":2}, {"x":21, "y":2}, {"x":0, "y":3}, {"x":1, "y":3}, {"x":2.5, "y":3, "w":1.25}, {"x":3.75, "y":3}, {"x":4.75, "y":3}, {"x":5.75, "y":3}, {"x":6.75, "y":3}, {"x":7.75, "y":3}, {"x":8.75, "y":3}, {"x":9.75, "y":3}, {"x":10.75, "y":3}, {"x":11.75, "y":3}, {"x":12.75, "y":3}, {"x":13.75, "y":3}, {"x":14.75, "y":3, "w":1.75}, {"x":16.5, "y":3}, {"x":18, "y":3}, {"x":19, "y":3}, {"x":20, "y":3}, {"x":0, "y":4}, {"x":1, "y":4}, {"x":2.5, "y":4, "w":1.5}, {"x":5, "y":4, "w":1.5}, {"x":6.5, "y":4, "w":7}, {"x":13.5, "y":4, "w":1.5}, {"x":16, "y":4, "w":1.5}, {"x":18, "y":4}, {"x":19, "y":4}, {"x":20, "y":4}, {"x":21, "y":3, "h":2}]
```

File: keyboards/maartenwut/mx5160/readme.md [hunk 9]
```diff
@@ -0,0 +1,15 @@
+# mx5160
+
+![mx5160](https://i.imgur.com/XZ9xUeq.jpg)
+
+## Support
+Keyboard Maintainer: [Maarten Dekkers](https://github.com/maartenwut)
```
Comment (noroadsleft) [inline keyboards/maartenwut/mx5160/readme.md]: Template was updated for this section.

```suggestion
* Keyboard Maintainer: [Maarten Dekkers](https://github.com/maartenwut)
```

File: keyboards/maartenwut/mx5160/readme.md [hunk 10]
```diff
@@ -0,0 +1,15 @@
+# mx5160
+
+![mx5160](https://i.imgur.com/XZ9xUeq.jpg)
+
+## Support
+Keyboard Maintainer: [Maarten Dekkers](https://github.com/maartenwut)  
+Hardware Supported: mx-5160 (Chicony 5160c case)
```
Comment (noroadsleft) [inline keyboards/maartenwut/mx5160/readme.md]:
```suggestion
* Hardware Supported: mx-5160 (Chicony 5160c case)
```

File: keyboards/maartenwut/mx5160/readme.md [hunk 11]
```diff
@@ -0,0 +1,15 @@
+# mx5160
+
+![mx5160](https://i.imgur.com/XZ9xUeq.jpg)
+
+## Support
+Keyboard Maintainer: [Maarten Dekkers](https://github.com/maartenwut)  
+Hardware Supported: mx-5160 (Chicony 5160c case)  
+Hardware Availability: [Github](https://github.com/Maartenwut/mx-5160)
```
Comment (noroadsleft) [inline keyboards/maartenwut/mx5160/readme.md]:
```suggestion
* Hardware Availability: [Github](https://github.com/Maartenwut/mx-5160)
```

**Comment by Maartenwut** [comment]
Could this be opened again?

File: keyboards/maartenwut/mx5160/info.json [hunk 12]
```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "mx5160", 
+    "url": "https://github.com/Maartenwut/mx-5160", 
+    "maintainer": "qmk", 
+    "width": 22, 
+    "height": 5, 
+    "layouts": {
+        "LAYOUT": {
```
Comment (noroadsleft) [inline keyboards/maartenwut/mx5160/info.json]: As the PCB supports more than one physical layout, this layout macro should be named `LAYOUT_all`.

```suggestion
        "LAYOUT_all": {
```

File: keyboards/maartenwut/mx5160/keymaps/default/keymap.c [hunk 13]
```diff
@@ -0,0 +1,42 @@
...
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+
+#define _MA 0 //Main layer
+#define _FN 1 //Main layer
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+[_MA] = LAYOUT(
```
Comment (noroadsleft) [inline keyboards/maartenwut/mx5160/keymaps/default/keymap.c]:
```suggestion
[_MA] = LAYOUT_all(
```

File: keyboards/maartenwut/mx5160/keymaps/default/keymap.c [hunk 14]
```diff
@@ -0,0 +1,42 @@
...
+    KC_F1,   KC_F2,   KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,     KC_8,     KC_9,      KC_0,      KC_MINS, KC_EQL,  _______, KC_BSPC,    KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS,    \
+    KC_F3,   KC_F4,   KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,     KC_I,     KC_O,      KC_P,      KC_LBRC, KC_RBRC,          KC_BSLS,    KC_P7,   KC_P8,   KC_P9,   _______,    \
+    KC_F5,   KC_F6,   KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,     KC_K,     KC_L,      KC_SCLN,   KC_QUOT, _______,          KC_ENT,     KC_P4,   KC_P5,   KC_P6,   KC_PPLS,    \
+    KC_F7,   KC_F8,   KC_LSFT, _______, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,     KC_M,     KC_COMM,   KC_DOT,    KC_SLSH, KC_RSFT,          _______,    KC_P1,   KC_P2,   KC_P3,               \
+    KC_F9,   KC_F10,  KC_LCTL,          KC_LALT,                            KC_SPC,                                            KC_RALT,                    MO(_FN),   KC_P0,   _______, KC_PDOT, KC_PENT),   \
+	
+[_FN] = LAYOUT(
```
Comment (noroadsleft) [inline keyboards/maartenwut/mx5160/keymaps/default/keymap.c]:
```suggestion
[_FN] = LAYOUT_all(
```

File: keyboards/maartenwut/mx5160/mx5160.h [hunk 15]
```diff
@@ -0,0 +1,47 @@
...
+
+/* This a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+#define LAYOUT( \
```
Comment (noroadsleft) [inline keyboards/maartenwut/mx5160/mx5160.h]:
```suggestion
#define LAYOUT_all( \
```

File: keyboards/maartenwut/mx5160/mx5160.c [hunk 16]
```diff
@@ -0,0 +1,71 @@
...
+        writePinHigh(B2);
+    }
+	
+    if (usb_led & (1 << USB_LED_SCROLL_LOCK)) {
+        writePinLow(B3);
+    } else {
+        writePinHigh(B3);
+    }
+  led_set_user(usb_led);
+}
```
Comment (noroadsleft) [inline keyboards/maartenwut/mx5160/mx5160.c]: Lines 50-71 can be replaced with:

```c
bool led_update_kb(led_t led_state) {
    bool res = led_update_user(led_state);
    if (res) {
        writePin(B2, !led_state.num_lock);
        writePin(B1, !led_state.caps_lock);
        writePin(B3, !led_state.scroll_lock);
    }
    return res;
}
```

[Customizing Functionality: LED Control - QMK Firmware Docs](https://docs.qmk.fm/#/custom_quantum_functions?id=led-control)

File: keyboards/maartenwut/mx5160/info.json (hunk 12 above)
Comment (noroadsleft) [inline keyboards/maartenwut/mx5160/info.json]: The layout name is actually here:

```suggestion
        "LAYOUT_all": {
```

File: keyboards/maartenwut/mx5160/info.json [hunk 17]
```diff
@@ -0,0 +1,12 @@
...
..., {"x":10, "y":1}, {"x":11, "y":1}, {"x":12, "y":1}, {"x":13, "y":1}, {"x":14, "y":1}, {"x":15, "y":1}, {"x":16, "y":1, "w":1.5}, {"x":18, "y":1}, {"x":19, "y":1}, {"x":20, "y":1}, {"x":21, "y":1}, {"x":0, "y":2}, {"x":1, "y":2}, {"x":2.5, "y":2, "w":1.75}, {"x":4.25, "y":2}, {"x":5.25, "y":2}, {"x":6.25, "y":2}, {"x":7.25, "y":2}, {"x":8.25, "y":2}, {"x":9.25, "y":2}, {"x":10.25, "y":2}, {"x":11.25, "y":2}, {"x":12.25, "y":2}, {"x":13.25, "y":2}, {"x":14.25, "y":2}, {"x":15.25, "y":2}, {"x":16.25, "y":2, "w":1.25}, {"x":18, "y":2}, {"x":19, "y":2}, {"x":20, "y":2}, {"x":21, "y":2}, {"x":0, "y":3}, {"x":1, "y":3}, {"x":2.5, "y":3, "w":1.25}, {"x":3.75, "y":3}, {"x":4.75, "y":3}, {"x":5.75, "y":3}, {"x":6.75, "y":3}, {"x":7.75, "y":3}, {"x":8.75, "y":3}, {"x":9.75, "y":3}, {"x":10.75, "y":3}, {"x":11.75, "y":3}, {"x":12.75, "y":3}, {"x":13.75, "y":3}, {"x":14.75, "y":3, "w":1.75}, {"x":16.5, "y":3}, {"x":18, "y":3}, {"x":19, "y":3}, {"x":20, "y":3}, {"x":0, "y":4}, {"x":1, "y":4}, {"x":2.5, "y":4, "w":1.5}, {"x":5, "y":4, "w":1.5}, {"x":6.5, "y":4, "w":7}, {"x":13.5, "y":4, "w":1.5}, {"x":16, "y":4, "w":1.5}, {"x":18, "y":4}, {"x":19, "y":4}, {"x":20, "y":4}, {"x":21, "y":3, "h":2}]
```
Comment (noroadsleft) [inline keyboards/maartenwut/mx5160/info.json]: ... and this JSON key should always be `layout`, in lowercase.

```suggestion
            "layout": [{"x":0, "y":0}, {"x":1, "y":0}, {"x":2.5, "y":0}, {"x":3.5, "y":0}, {"x":4.5, "y":0}, {"x":5.5, "y":0}, {"x":6.5, "y":0}, {"x":7.5, "y":0}, {"x":8.5, "y":0}, {"x":9.5, "y":0}, {"x":10.5, "y":0}, {"x":11.5, "y":0}, {"x":12.5, "y":0}, {"x":13.5, "y":0}, {"x":14.5, "y":0}, {"x":15.5, "y":0}, {"x":16.5, "y":0}, {"x":18, "y":0}, {"x":19, "y":0}, {"x":20, "y":0}, {"x":21, "y":0}, {"x":0, "y":1}, {"x":1, "y":1}, {"x":2.5, "y":1, "w":1.5}, {"x":4, "y":1}, {"x":5, "y":1}, {"x":6, "y":1}, {"x":7, "y":1}, {"x":8, "y":1}, {"x":9, "y":1}, {"x":10, "y":1}, {"x":11, "y":1}, {"x":12, "y":1}, {"x":13, "y":1}, {"x":14, "y":1}, {"x":15, "y":1}, {"x":16, "y":1, "w":1.5}, {"x":18, "y":1}, {"x":19, "y":1}, {"x":20, "y":1}, {"x":21, "y":1}, {"x":0, "y":2}, {"x":1, "y":2}, {"x":2.5, "y":2, "w":1.75}, {"x":4.25, "y":2}, {"x":5.25, "y":2}, {"x":6.25, "y":2}, {"x":7.25, "y":2}, {"x":8.25, "y":2}, {"x":9.25, "y":2}, {"x":10.25, "y":2}, {"x":11.25, "y":2}, {"x":12.25, "y":2}, {"x":13.25, "y":2}, {"x":14.25, "y":2}, {"x":15.25, "y":2}, {"x":16.25, "y":2, "w":1.25}, {"x":18, "y":2}, {"x":19, "y":2}, {"x":20, "y":2}, {"x":21, "y":2}, {"x":0, "y":3}, {"x":1, "y":3}, {"x":2.5, "y":3, "w":1.25}, {"x":3.75, "y":3}, {"x":4.75, "y":3}, {"x":5.75, "y":3}, {"x":6.75, "y":3}, {"x":7.75, "y":3}, {"x":8.75, "y":3}, {"x":9.75, "y":3}, {"x":10.75, "y":3}, {"x":11.75, "y":3}, {"x":12.75, "y":3}, {"x":13.75, "y":3}, {"x":14.75, "y":3, "w":1.75}, {"x":16.5, "y":3}, {"x":18, "y":3}, {"x":19, "y":3}, {"x":20, "y":3}, {"x":0, "y":4}, {"x":1, "y":4}, {"x":2.5, "y":4, "w":1.5}, {"x":5, "y":4, "w":1.5}, {"x":6.5, "y":4, "w":7}, {"x":13.5, "y":4, "w":1.5}, {"x":16, "y":4, "w":1.5}, {"x":18, "y":4}, {"x":19, "y":4}, {"x":20, "y":4}, {"x":21, "y":3, "h":2}]
```

File: keyboards/maartenwut/mx5160/rules.mk [hunk 18]
```diff
@@ -0,0 +1,32 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   atmega32a    bootloadHID
```
Comment (fauxpark) [inline keyboards/maartenwut/mx5160/rules.mk]:
```suggestion
#   ATmega32A    bootloadHID
#   ATmega328P   USBasp
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**